### PR TITLE
Find me a home

### PR DESCRIPTION
**Pull request by sitapriyamoorthi**
#### What changes are being implemented in this Pull Request?

Porting content from WDL 101 with things about Cromwell at Fred Hutch specifications for WDL. 

#### What was your approach?

Copied over the content from WDL 101

### Tell potential reviewers what kind of feedback you are soliciting.

See where this can fit!

### New Content Checklist

- [ ] New content/chapter is in an Rmd file with [this kind of format and headers](https://github.com/jhudsl/OTTR_Template/blob/main/02-chapter_of_course.Rmd).

- [ ] New content/chapter contains learning objectives.

- [ ] [Bookdown successfully re-renders and any new content files have been added to the _bookdown.yml](https://github.com/jhudsl/OTTR_Template/wiki/Publishing-with-Bookdown).

- [ ] [Spell check runs successfully](https://www.ottrproject.org/customize-robots.html#Spell_checking)).

- [ ] Any newly necessary packages that are needed have been added to the [Dockerfile and image](https://www.ottrproject.org/customize-docker.html).

- [ ] Images are in the [correct format for rendering](https://www.ottrproject.org/writing_content.html#set-up-images).

- [ ] Every new image has [alt text and is in a Google Slide](https://www.ottrproject.org/writing_content.html#Accessibility).

- [ ] Each slide is described in the notes of the slide so learners relying on a screen reader can access the content. See https://lastcallmedia.com/blog/accessible-comics for more guidance on this.

- [ ] The color palette choices of the slide are contrasted in a way that is friendly to those with color vision deficiencies.
You can check this using [Color Oracle](https://colororacle.org/).

### CODE REVIEW

File: 01-intro.Rmd [hunk 1]
```diff
@@ -314,3 +314,75 @@ check JSON input; unknown input/output: greetings.username
...
+At Fred Hutch we have huge array of pre-curated software modules installed on our SLURM cluster which you can [read about in SciWiki](https://sciwiki.fredhutch.org/scicomputing/compute_scientificSoftware/).   The custom configuration of our Cromwell server allows users to specify one or more modules to use for individual tasks in a workflow.  The desired module(s) can be requested for a task in the `runtime` block of your calls like this:
+
+```
+runtime {
+  modules: "GATK/4.2.6.1-GCCcore-11.2.0 SAMtools/1.16.1-GCC-11.2.0"
+}
+```
+
+In this example, we specify two modules, separated by a space (with quotes surrounding them). The GATK module will be loaded first, followed by the SAMtools module.   In this example you'll note the "toolchain" used to build each modules is the same ("GCC-11.2.0").  When you load >1 module for a single task it is important to ensure that they are compatible with each other.  Choose versions built with the same toolchain if you can.  
+
```
Comment (caalo) [inline 01-intro.Rmd:382]: I feel like these are the most relevant information, as the other un-highlighted part of the commit has been covered by other parts in the guide already. I would put this in the backend appendix that Ash is writing and have a toggle to hide it as its own section.

Comment (sitapriyamoorthi) [reply 01-intro.Rmd:382]: okay. also i did just want to add this part alone to the intro not to main! cause main has added parts!

**Comment by caalo** [comment]
Did some merging (merged main into intro, then resolved conflicts between intro and find me a home) to make this PR only about one file change.

**Comment by caalo** [comment]
@aofarrel I think the stuff I highlighted in the PR review may be more relevant in the backends appendix.

**Comment by caalo** [comment]
Moved relevant info to the appendix in this PR: https://github.com/fhdsl/WDL_Workflows_Guide/pull/46. Going to close this PR out.

**Pull request closed without merging**